billit/billit.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+from constants import SANDBOX_BASE_URL
+
+load_dotenv()
+
+
+class Client:
+    """
+    Base client for all API clients
+    """
+
+    def __init__(self, api_key):
+        self.api_key = api_key or os.environ.get("BILLIT_API_KEY")
+        self.base_url = SANDBOX_BASE_URL
```

Comment:
Let's

- [ ] Add a base URL parameter, defaulting to `SANDBOX_BASE_URL`
- [ ] Not use environment variables in here and just keep them for our personal testing files.

```suggestion
    def __init__(self, api_key, base_url=SANDBOX_BASE_URL):
        self.api_key = api_key
        self.base_url = base_url
```